src/core/main.c

```diff
@@ -698,6 +698,19 @@ static int parse_config_file(void) {
 
         config_parse_many(fn, conf_dirs_nulstr, "Manager\0", config_item_table_lookup, items, false, NULL);
 
+        _cleanup_free_ char *rvalue = NULL;
```

Comment:
Yes, this belongs among the rest of the kernel cmdline code in parse_proc_cmdline_item().